README.adoc

```diff
@@ -92,6 +92,21 @@ $ ./run.py
 
 === Test-only dependencies
 
+.Install all package dependencies at once (Fedora)
+[source, shell script]
+----
+dnf install curl nmap-ncat
+----
+
+.Install all Python test dependencies at once
```

Comment:
From https://github.com/skupperproject/skupper-router/pull/407#pullrequestreview-958649989 Just a couple of minor suggestions -
Just say Install package dependencies instead of Install all package dependencies at once
and say Install Python test dependencies from requirements-dev.txt instead of Install all Python test dependencies at once

Reply:
@ganeshmurthy done